catalog/controller/module/tawkto.php

```diff
@@ -31,13 +31,18 @@ public function __construct($registry) {
 	 */
 	public function index()
 	{
+		if (session_status() === PHP_SESSION_NONE && !headers_sent()) {
```

Comment:
Same as with other implementations, only check/start the session if secure mode is enabled and API key is set. Otherwise we incur the session overhead w/o any benefit